test/calendar.ts

```diff
@@ -0,0 +1,97 @@
+import {assert} from 'mocha-webdriver';
+import {getGrist} from "./getGrist";
+
+//not to pretty way to get events from currently used calendar control. but it's working. 
+function getAbstractFromCalendarObject(eventId: number) {
+    return `
```

Comment:
This is basically going to `executeScript`, so it would be nice if it could be formatted as a function.

Again, the use of `this` makes me nervous.

Reply:
Maybe you could add a helper function into the widget to make this shorter. if you do, the Grist convention is just to have the function name start with `test`.